src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+const DAILY_RATE = 40;
+const LONG_TERM_RENTAL_DAYS = 7;
+const LONG_TERM_DISCOUNT = 50;
+const MID_TERM_RENTAL_DAYS = 3;
+const MID_TERM_DISCOUNT = 20;
+
+const baseCost = numberOfDays * DAILY_RATE;
```

Comment:
The variable `numberOfDays` is not defined. You should use the parameter `days` instead to calculate the `baseCost`.